index.css

```diff
@@ -0,0 +1,186 @@
+body {
+    background-color: #f2f2f2;
+    color: #333;
+    font-weight: 400;
+    line-height: 1.3;
+    margin: 32px;
+    font-family: Arial, Arial, Helvetica, sans-serif;
+}
+
+img {
+    max-width: 100%;
+}
+
+.nav {
+    box-sizing: border;
+    padding: 15px;
+}
+
+.topnav {
+    list-style-type: none;
+    margin: 0;
+    padding: 0;
+    overflow: hidden;
+    background-color: #333;
+}
+
+.topnav a {
+    float: left;
+    color: #f2f2f2;
+    text-align: center;
+    padding: 14px 16px;
+    text-decoration: none;
+    font-size: 17px;
+}
+
+.topnav a:hover {
+    background-color: #ddd;
+    color: black;
+}
+
+.u-list-reset {
+    list-style: none;
+    margin-top: 0;
+    margin-bottom: 0;
+    padding-left: 0;
+}
+
+.title {
+    box-sizing: border;
+    padding: 10px;
+    margin-top: -30px;
+    margin-bottom: -10px;
```

Comment:
You're relying on negative margins here to fix what should instead be addressed by undoing the default margins on the `h1` element. Also...you're adding 10px of padding to the top and bottom, then removing it with the margins. Even if this gets you the look you want, there are simpler, more straight forward ways to do so and you should work toward those.